etna/whatson/models.py

```diff
@@ -0,0 +1,618 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.images import get_image_model_string
+from wagtail.models import Orderable
+from wagtail.search import index
+from wagtail.snippets.models import register_snippet
+
+from etna.articles.models import ArticleTagMixin
+from etna.collections.models import TopicalPageMixin
+from etna.core.models import BasePageWithIntro
+
+
+class BookingType(models.TextChoices):
+    """
+    This model is used for the Eventbrite booking type.
+    TODO: Find out what the other booking types are from Eventbrite.
+    """
+
+    DROP_IN = "drop_in", _("Drop in")
+    BOOKING = "booking", _("Booking")
```

Comment:
Based on [scanning the docs](https://www.eventbrite.com/platform/api#/reference/event-teams/create/retrieve-an-event), I can't think what this would map to in Eventbrite. It feels like something we've invented. Maybe this is just something we should work out in a property method or something, depending on whether there is any booking option data saved?

Reply:
You're right, it most likely is something we've invented. It'd be good to have this as an option though, so I'm guessing it's down to editor-discretion. I think I may leave it out for now, then have a chat with everyone? What do you think?

Reply:
Yep - this was never going to be the end of the work anyway... I think it makes sense to take time and be more certain about it